_component_design/tooltip.md

```diff
@@ -13,28 +13,304 @@ last-modified: 2017-08-17
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-large-tooltips.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+## Loading state
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Loading states are rarely found when using the tooltip pattern. The loading state is applied exclusively when using data visualizations.
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Loading State."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-loading.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+
+## Interaction behaviors
+{: .hxSectionTitle}
+
+Tooltips can be activated via either a hover or a click on the trigger element.
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Hover interaction
+{: .hxSubSectionTitle}
+
+
+Hover tooltips are best used on large breakpoints and desktop designs. Tooltips can have both hover and click interactions on the same tooltip across breakpoints. Hover enabled Tooltips go through the following interaction lifecycle:
+ 
+- **Hidden:** The base state of a tooltip is to be hidden until the trigger is engaged by the pointed
+- **Displayed:** Tooltips enter the display state once the pointer has hovered the trigger element for 500ms.
+- **Persist:**  While a pointer is hovered inside a tooltip it will persist indefintely to allow for user consumption.
+- **Removal:** After the pointer leaves either the trigger or the tooltip the tooltip is ready for removal. After 1600ms the tip is removed.
+
+
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Tooltip hover lifecycle."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-hover.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Click interaction
+{: .hxSubSectionTitle}
+
+Click enabled tooltips are best used for smaller breakpoint and mobile designs. Tooltips can have both hover and click interactions on the same tooltip across breakpoints. Click enabled tooltips go through the following interaction lifecycle 
+
+- **Hidden:** The base state of a tooltip is to be hidden until the trigger is clicked by the pointer
+- **Displayed:** Tooltips enter the displayed state once the pointer has clicked the trigger element.
+- **Persist:**  Clickable tooltips persist until manually removed by a user.
+ - **Removal:** Tooltips reenter the hidden state once the pointer has either: 
+
+	- reclicked the trigger element.
```

Comment:
- Re-clicked ... 
- Clicked outside the tooltip.

Reply:
updated.